packages/orbit-themer/package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "@kiwicom/orbit-themer",
-  "version": "0.0.3",
+  "version": "1.0.0",
   "private": "true",
   "description": "Design your own orbit-components theme with custom colors",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "dev": "webpack-dev-server --mode development --open --hot --host 0.0.0.0",
-    "flow": "flow",
```

Comment:
could you rewrite it to a typescript? 

simply rewrite files from `jsx` to `tsx` and add tsconfig, build it with babel.

Reply:
or you can also check how we use `tsup` inside `orbit-tracking` or `orbit-design-tokens` and also use it ;)

Reply:
Done